Packet++/src/PacketUtils.cpp

```diff
@@ -35,7 +35,10 @@ uint16_t computeChecksum(ScalarBuffer<uint16_t> vec[], size_t vecSize)
 			PCPP_LOG_DEBUG("1 byte left, adding value: 0x" << std::uppercase << std::hex << lastByte);
 			// We have read the latest byte manually but this byte should be interpreted
 			// as a 0xFF on LE and a 0xFF00 on BE to have a proper checksum computation
-			localSum += le16toh(lastByte);
+			uint16_t lastSum;
+			memcpy(&lastSum, &lastByte, 1);

```

Comment:
@seladb proposition from @guebe, IMO the le16toh() is more explicit.

Which one do you prefer?